Testing/CMakeLists.txt

```diff
@@ -0,0 +1,88 @@
+cmake_minimum_required(VERSION 3.18)
+project(unit_testing)
+
+set(CMAKE_C_COMPILER "gcc")
+set(CMAKE_CXX_COMPILER "g++")
+
+# GoogleTest requires at least C++14
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# Locate GTest
+find_package(GTest REQUIRED)
```

Comment:
Could you real quick make a 'unit testing for zp3.5' doc under ZeroPilot 3.5 on confluence. You dont need to add anything to it yet, but maybe just a link to where gtest can be installed?

Reply:
done